package.json

```
@@ -70,7 +70,7 @@
     "yargs-parser": "10.x"
   },
   "peerDependencies": {
-    "jest": ">=22 <24"
+    "jest": ">=22"
```

Considering the changes to babel-core (bridge) in this PR, it is safe to say it won't work with Jest `<24.`
Also considering that Jest has changed quite a bit between versions, it doesn't hurt to stay conservative and still explicitly require `<25` IMHO.

```suggestion
    "jest": ">=24 <25"
```